auth/src/AzureDevOpsSubscriptionProvider.ts

```diff
@@ -183,8 +183,8 @@ export class AzureDevOpsSubscriptionProvider implements AzureSubscriptionProvide
         }
     }
 
-    public onDidSignIn: Event<void> = () => { return new Disposable(() => { /*empty*/ }) };
-    public onDidSignOut: Event<void> = () => { return new Disposable(() => { /*empty*/ }) };
+    public onDidSignIn: vscode.Event<void> = () => { return { dispose(): void { /*empty*/ } }; };
```

Comment:
Is there a difference between using the disposal and just returning `dispose()`?